ai-sample/SampleAIWorkflows/aisearch_index.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
I would keep this out side of the workflow project - maybe under new folder (e.g., deployment)